packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -2052,23 +2096,23 @@ describe('plugin-meetings', () => {
                 assert.notCalled(locusMediaRequestStub);
               });
 
-              it(`unpublishTracks() call while media is ${mediaEnabled ? 'enabled' : 'disabled'}`, async () => {
+              it(`unpublishStreams() call while media is ${mediaEnabled ? 'enabled' : 'disabled'}`, async () => {
                 await meeting.addMedia({audioEnabled: mediaEnabled});
                 await simulateRoapOffer();
-                await meeting.publishTracks({microphone: fakeMicrophoneTrack});
+                await meeting.publishStreams({microphone: fakeMicrophoneStream});
                 await stableState();
 
                 resetHistory();
 
-                await meeting.unpublishTracks([fakeMicrophoneTrack]);
+                await meeting.unpublishStreams([fakeMicrophoneStream]);
                 await stableState();
 
                 // the roap media connection should be updated
                 if (isMultistream) {
-                  assert.calledOnceWithExactly(fakeMultistreamRoapMediaConnection.unpublishTrack, fakeMicrophoneTrack);
+                  assert.calledOnceWithExactly(meeting.sendSlotManager.unpublishStream, MediaType.AudioMain);
```

Comment:
ideally we should be checking that unpublishStream on the right sendSlot is called here, not that sendSlotManager is called

Reply:
We do check if the unpublishStream is called with the right MediaType in this line itself.

As already mentioned in one of the previous comments, sendSlotManger has its own tests to know if the methods of the right sendSlot is being called. Do we still need to stub one step deeper?

Reply:
I see you've now changed it all to use SendSlotManager.getSlot() and check the right method is called on it, I think that's fine, so marking this as resolved.